docker/terraform/keycloak.tf

```diff
@@ -68,10 +68,29 @@ resource "keycloak_openid_client" "openid_client" {
     "http://vault.${var.container_domain}:8200/ui/vault/auth/oidc/oidc/callback"
   ]
 }
-resource "keycloak_generic_client_protocol_mapper" "groups" {
+
+# Add Vault as OpenID Connect Client for JWT
+resource "keycloak_openid_client" "jwt_client" {
+  realm_id  = keycloak_realm.realm.id
+  client_id = "hashicorp-vault-jwt"
+
+  name    = "HashiCorp Vault vault.${var.container_domain}"
+  enabled = true
+
+  access_type           = "PUBLIC"
```

Comment:
similar to my comments above, as a newbie I'm not quite sure yet why to use `PUBLIC/CONFIDENTIAL` here and for which "flow" (https://registry.terraform.io/providers/devkoriel/keycloak/latest/docs/resources/openid_client#access_type). Would love to see a small line of comment or something in the Usage docs for that.